proposals/lip-0051.md

```diff
@@ -1997,20 +2008,15 @@ def payMessageFee(payFromAddress: Address, fee: uint64, receivingChainID: ChainI
     messageFeeTokenID = Interoperability.getMessageFeeTokenID(receivingChainID)
 
     # Sender should have enough balance to pay the message fees.
-    if availableBalance(payFromAddress, messageFeeTokenID) < fee:
+    if getAvailableBalance(payFromAddress, messageFeeTokenID) < fee:
         raise Exception("Insufficient balance for the message fee.")
 
     # Pay the fee.
     availableBalance(payFromAddress, messageFeeTokenID) -= fee
 
     # If the chain is the fee token native chain, escrow.
     if getChainID(messageFeeTokenID) == OWN_CHAIN_ID:
-        if there exists an entry escrowStore(receivingChainID, messageFeeTokenID) in the escrow substore:
-            escrowAmount(receivingChainID, messageFeeTokenID) += fee
-        else:
-            create an escrow substore entry with
-                key = receivingChainID + messageFeeTokenID
-                value = encode(escrowStoreSchema, {"amount": fee})        
+        escrowAmount(receivingChainID, messageFeeTokenID) += fee
```

Comment:
I would add comment explaining that the escrow is created at chain registration, so it should exist

Reply:
Done.